app/assets/stylesheets/thredded/components/_currently-online.scss

```diff
@@ -6,36 +10,32 @@
 
   &.is-expanded {
     @include position(fixed, null 0 0 null);
-
-    .currently-online--users {
-      height: auto;
-      visibility: visible;
-    }
   }
 
-  @include media($mobile) {
-    display: none;
+  &--title {
+    margin-bottom: $base-spacing;
   }
-}
 
-.currently-online--title {
-  margin-bottom: $base-spacing;
-}
+  &--avatar {
+    @include size(rem(28));
+    background-color: $base-font-color;
+    border-radius: 50%;
+    display: inline-block;
+    margin-right: $small-spacing;
+    vertical-align: middle;
+  }
 
-.currently-online--avatar {
-  @include size(rem(28));
-  background-color: $base-font-color;
-  border-radius: 50%;
-  display: inline-block;
-  margin-right: $small-spacing;
-  vertical-align: middle;
-}
+  &--user {
+    padding: $small-spacing 0;
+  }
 
-.currently-online--user {
-  padding: $small-spacing 0;
-}
+  &--users {
+    height: 0;
+    visibility: hidden;
+  }
 
-.currently-online--users {
-  height: 0;
-  visibility: hidden;
+  &.is-expanded &--users {
```

Comment:
It seems to me that having the two states of the same component next to each other is more readable.